pkg/services/sqlstore/migrations/stats_mig.go

```diff
@@ -2,38 +2,6 @@ package migrations
 
 import . "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 
-func addStatsMigrations(mg *Migrator) {
```

Comment:
Comment this code instead of deleting it. It's being used in some development but not released yet.

Reply:
@bergquist done, PTAL